python-lib/sharepoint_client.py

```diff
@@ -447,15 +469,17 @@ def __init__(self, sharepoint_user_name, sharepoint_password, sharepoint_tenant,
 
     def get(self, url, headers=None, params=None):
         headers = {} if headers is None else headers
```

Comment:
Nitpick: A syntax I prefere to use is : 
```suggestion
        headers = headers or {}
```